systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaUtils.java

```diff
@@ -151,4 +154,79 @@ public static void waitForClusterStability(String clusterName) {
             return false;
         });
     }
+
+    /**
+     * Method which, update/replace Kafka configuration
+     * @param clusterName name of the cluster where Kafka resource can be found
+     * @param kafkaDynamicConfiguration enum instance, which defines all supported configurations
+     * @param value value of specific property
+     */
+    public static void updateSpecificConfiguration(String clusterName, String kafkaDynamicConfiguration, Object value) {
+        KafkaResource.replaceKafkaResource(clusterName, kafka -> {
+            LOGGER.info("Kafka config before updating '{}'", kafka.getSpec().getKafka().getConfig().toString());
+            Map<String, Object> config = kafka.getSpec().getKafka().getConfig();
+            config.put(kafkaDynamicConfiguration, value);
+            kafka.getSpec().getKafka().setConfig(config);
+            LOGGER.info("Kafka config after updating '{}'", kafka.getSpec().getKafka().getConfig().toString());
+        });
+    }
+
+    /**
+     * Method which, extends the @see updateConfiguration(String clusterName, KafkaConfiguration kafkaConfiguration, Object value) method
+     * with stability and ensures after update of Kafka resource there will be not rolling update
+     * @param clusterName name of the cluster where Kafka resource can be found
+     * @param kafkaDynamicConfiguration key of specific property
+     * @param value value of specific property
+     */
+    public static void  updateConfigurationWithStabilityWait(String clusterName, String kafkaDynamicConfiguration, Object value) {
+        updateSpecificConfiguration(clusterName, kafkaDynamicConfiguration, value);
+        PodUtils.verifyThatRunningPodsAreStable(KafkaResources.kafkaStatefulSetName(clusterName));
+    }
+
+    /**
+     * Method, verifying that updating configuration were successfully changed inside Kafka CR
+     * @param kafkaDynamicConfiguration key of specific property
```

Comment:
For all these I would call it `brokerConfigName` or something. You don't care, in these methods whether the particular config supports dynamic update or not.